modules/@apostrophecms/ui/ui/apos/components/AposSlat.vue

```diff
@@ -71,6 +73,14 @@ export default {
     engaged: {
       type: Boolean,
       default: false
+    },
+    displayDragIcon: {
+      type: Boolean,
+      default: true
```

Comment:
The entire slat is draggable so this wouldn't be enough. I would have this option be called `sortable` and when false opt out of the entire sortablejs dance